test/integration/run.sh

```diff
@@ -17,12 +17,7 @@ fi
 export CLICKHOUSE_BACKUP_BIN="$(pwd)/clickhouse-backup/clickhouse-backup-race"
 export LOG_LEVEL=${LOG_LEVEL:-info}
 
-if [[ -f "${CUR_DIR}/credentials.json" ]]; then
```

Comment:
Please rollback these changes

we still need to use credentials.json for test on real GCS bucket
export GSC_TESTS=${GCS_TESTS:-}
means export empty value if variable not already present

Reply:
Reverted